app/views/endemics/guidance/apply-guidance-for-farmers.njk

```diff
@@ -197,11 +197,11 @@
             <p>You must agree with the vet what they’ll focus on. Ask them to read the <a href="#">[vet guidance].</a>
             </p>
             <h2 id="after-the-review">What happens on a review or follow-up</h2>
-            <p>Read the guidance about <a href="../farmer/guidance-visit.html">what happens on a review or follow-up.</a>
+            <p>Read the guidance about <a href="https://fcp-ahwr-prototype.herokuapp.com/v26/farmer/guidance-visit">what happens on a review or follow-up.</a>
```

Comment:
Why those are prototype links? just use #, if they are not confirmed yet

Reply:
> Why those are prototype links? just use #, if they are not confirmed yet

done